src/routes/index.js

```diff
@@ -4,11 +4,10 @@ import userRouter from "./user/index.js";
 import MoneybookRouter from "./moneybook/index.js";
 import MoneybookDetailRouter from "./moneybook_detail/index.js";
 
-router.use("/user", userRouter);
-router.use("/moneybooks", MoneybookRouter);
-router.use("/moneybooks/moneybook_idx", MoneybookRouter);
-router.use("/moneybooks", MoneybookDetailRouter);
-router.use("/moneybooks/:idx", MoneybookDetailRouter);
+router.use("/users", userRouter);
+router.use("/moneybook", MoneybookRouter);
```

Comment:
```suggestion
router.use("/moneybook", MoneybookRouter);
```
/moneybooks가 좋아보입니다.